mmrm.Rproj

```diff
@@ -16,7 +16,6 @@ AutoAppendNewline: Yes
 StripTrailingWhitespace: Yes
 
 BuildType: Package
```

Comment:
suggest to not modify

Reply:
This has been restored to the version on main